### ISSUE TITLE
Test Suite

### ISSUE THREAD

**Issue by Trineos** [tsaglam/Carcassonne#7]
Does there exist a test suite for this project?

**Comment by tsaglam** [comment]
No, sadly, I never really wrote many tests here. From manual testing, the program behaves as it should. The build is only failing due to sonar. May I ask why you are interested in tests?
